e220/config.go

```diff
@@ -0,0 +1,380 @@
+// Package e220 implements a driver for the e220 LoRa module.
+package e220
+
+import "fmt"
+
+// Config represents E220's configuration parameters
+type Config struct {
+	ModuleAddr         uint16
+	UartSerialPortRate uint8
+	AirDataRate        uint8
+	SubPacket          uint8
+	RssiAmbient        uint8
+	TransmitPower      uint8
+	Channel            uint8
+	RssiByte           uint8
+	TransmitMethod     uint8
+	WorCycleSetting    uint8
+	EncryptionKey      uint16
+	Version            uint8
+	errors             []error
+}
+
+func (c *Config) paramsToBytes(
+	bytes *[]byte,
+) error {
+	// byte [8] is read only
+	if len(*bytes) < E220RegisterLength-1 {
+		return fmt.Errorf("length of bytes must be greater than or equal to %d: got=%d", E220RegisterLength-1, (*bytes))
+	}
+	(*bytes)[0] = byte((c.ModuleAddr & 0xFF00) >> 8)
+	(*bytes)[1] = byte((c.ModuleAddr & 0x00FF) >> 0)
+	(*bytes)[2] = byte(((c.UartSerialPortRate & 0x07) << 5) | (c.AirDataRate & 0x1F))
+	reserved := byte(0b000)
+	(*bytes)[3] = byte(((c.SubPacket & 0x03) << 6) | ((c.RssiAmbient & 0x01) << 5) | ((reserved & 0x07) << 2) | (c.TransmitPower & 0x03))
+	(*bytes)[4] = byte(c.Channel)
+	reserved = byte(0b000)
+	(*bytes)[5] = byte(((c.RssiByte & 0x01) << 7) | ((c.TransmitMethod & 0x01) << 6) | ((reserved & 0x07) << 3) | (c.WorCycleSetting & 0x07))
+	(*bytes)[6] = byte((c.EncryptionKey & 0xFF00) >> 8)
+	(*bytes)[7] = byte((c.EncryptionKey & 0x00FF) >> 0)
```

Comment:
No need to pass in a pointer to a slice. A slice is already composed of a pointer to the underlying array.  So you can change `(*bytes)[0] = byte(...)` to ``bytes[0] = byte(...)`  and `paramsToBytes(bytes *[]byte)` to `paramsToBytes(bytes []byte)` and you'll get same behaviour. 

Also one tends not to break function signatures with newlines in Go, especially if its a sufficiently small function signature.